src/nplinker/loader.py

```diff
@@ -171,17 +168,18 @@ def load(self):
         # or a complete failure to parse things, so bail out
         if len(self.strains) == 0:
             raise Exception(
-                f'Failed to find *ANY* strains, missing {STRAIN_MAPPINGS_FILENAME}?')
+                f'Failed to find *ANY* strains, missing {STRAIN_MAPPINGS_FILENAME}?'
+            )
 
         return True
 
     def _start_downloads(self):
-        self._downloader = PODPDownloader(self._platform_id)
-        self._root = self._downloader.project_file_cache
+        downloader = PODPDownloader(self._platform_id)
```

Comment:
Please modify tests/test_nplinker_local.py as well, in particular the test that checks for the hash of the json file (`npl` fixture)